lib/Zonemaster/Backend/Config.pm

```diff
@@ -58,22 +72,130 @@ sub BackendDBType {
     }
 }
 
-sub DB_user {
-    my ($self) = @_;
+=head2 MYSQL_database
+
+Returns the L<MYSQL.database|https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#mysql>
+property from the loaded config, or the L<DB.database_name|https://github.com/zonemaster/zonemaster-backend/blob/master/docs/Configuration.md#db>
```

Comment:
The fragments should be `#db-section`, it is even possible to point directly to the right section with `#database_name`